subjects/road_intersection/audit/README.md

```diff
@@ -1,86 +1,98 @@
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Right"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Right"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Right"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Right"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Right"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Right"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the key `"Right"`.
+##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the `"Right"` key.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate one vehicle for all the lanes (to this at least 3 times).
+##### Try to generate one vehicle for all the lanes (do this at least 3 times).
```

Comment:
What is the context of "lanes" here?
If each direction has two lanes, can it be possible to certainly know which lane will be used?
For example, perhaps a vehicle can go straight from either lane.

Reply:
lanes == traffic lanes
You can say that each road has two lanes in this program

Reply:
My comment is irrelevant. We're using single carriageways.